packages/elements/src/components/ino-checkbox/ino-checkbox.scss

```diff
@@ -4,54 +4,26 @@
 @use '@material/checkbox';
 @use 'base/theme';
 
-:host {
-  /**
-   * @prop --ino-checkbox-color: primary color of the checkbox
-   * @prop --ino-checkbox-color-hover: color of the checkbox on hover
-   * @prop --ino-checkbox-color-active: color of the checkbox if active
-   * @prop --checkbox-container-color-unchecked: color of the checkbox container if unchecked
-   * @prop --ino-checkbox-text-color-disabled: color of the checkbox text if disabled
-   * @prop --ino-checkbox-background-color-disabled: color of the checkbox background if disabled
-   * @prop --ino-checkbox-checkmark-color: primary color of the check mark
-   * @prop --ino-checkbox-checkmark-color-disabled: color of the check mark if disabled
-   */
-
-   --checkbox-color: var(--ino-checkbox-color, #{theme.color(primary)});
-   --checkbox-color-hover: var(
-     --ino-checkbox-color-hover,
-     #{theme.color(primary, light)}
-   );
-   --checkbox-color-active: var(
-     --ino-checkbox-color-active,
-     #{theme.color(primary, dark)}
-   );
-   --checkbox-container-color-unchecked: var(
-     --ino-checkbox-container-color-unchecked,
-     #{theme.color(primary, light)}
-   );
-   --checkbox-text-color-disabled: var(
-     --ino-checkbox-text-color-disabled,
-     #{theme.color(dark)}
-   );
-   --checkbox-background-color-disabled: var(
-     --ino-checkbox-background-color-disabled,
-     #{theme.color(light, light)}
-   );
-   --checkbox-checkmark-color: var(--ino-checkbox-checkmark-color, #fff);
-   --checkbox-checkmark-color-disabled: var(
-     --ino-checkbox-checkmark-color-disabled,
-     #{theme.color(light)}
-   );
-}
+$primary-dark: #2C02FF;
+$primary-base: #4B4EFF;
+$primary-light: #2D5DFF4D;
```

Comment:
Instead of strictly following the design, we might want to use our color set which is quite similar. You can find it here: https://github.com/inovex/elements/blob/master/packages/elements/src/components/base/_theme.scss

And use it like this:
`theme.color(primary)` or `theme.color(primary, light)`


This allows us to be flexible in the future, e.g. if we want to use completely different colors for all our components

Reply:
Thanks! I changed it to the theme colors, but primary light is currently a bit off. Should be fixed after we adjust the theme colors